invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordCommunitiesList.js

```diff
@@ -34,7 +33,7 @@ export class RecordCommunitiesList extends Component {
           </Placeholder.Header>
         </Placeholder>
       );
-    } else if (_isEmpty(communities) && permissions.can_manage) {
+    } else if (_isEmpty(communities)) {
```

Comment:
it was done on purpose, in order to hide the communities side panel if the user is not an owner, you need to alter the jinja part to check the same condition, to prevent the box from displaying if no communities are attached.

Reply:
I thought we agreed on having the empty message showing at all times? Or am I misunderstanding